packages/cli/e2e/__tests__/deploy.spec.ts

```diff
@@ -244,4 +244,16 @@ Update and Unchanged:
     expect(result.stderr).toContain('Failed to deploy your project. Unable to find constructs to deploy.')
     expect(result.status).toBe(1)
   })
+
+  it('Should deploy a project with snapshots', async () => {
+    const result = await runChecklyCli({
+      args: ['deploy', '--force'],
+      apiKey: config.get('apiKey'),
+      accountId: config.get('accountId'),
+      directory: path.join(__dirname, 'fixtures', 'snapshot-project'),
+      env: { PROJECT_LOGICAL_ID: projectLogicalId },
+    })
+    expect(result.status).toBe(0)
+    // TODO: Add assertions that the snapshots are successfully uploaded.
```

Comment:
We don't expose the `check.snapshots` from the public API, so we can't do a stronger assertion at the moment. Still, I think that it makes sense to put the basic test in place and make sure `checkly deploy` doesn't error.